### ISSUE TITLE
Max duration feature

### ISSUE THREAD

**Issue by Equin** [a914-gowtham/Android-video-trimmer#28]
Hi, can you say when this pull request (Added Max Duration #27) will be released ? 
And did i undesrstand it right. if i will set 10s max duration then users will be able to cut videos up to 10 sec and less ?

**Comment by a914-gowtham** [comment]
yeah, you are right. I need to check again the pull request. If everything is ok, I will make a release by 2 days

**Comment by a914-gowtham** [comment]
Min to a max feature already available in this lib.
`
TrimVideo.activity(videoUri)
          .setTrimType(TrimType.MIN_MAX_DURATION)
          .setMinToMax(10, 30)  //seconds
          .start(this);
`

**Comment by Equin** [comment]
@a914-gowtham, Oh, i see, but can it be possible to lock that area like with min duration? Right now (min set 10s, max set 5 mins.), when i am moving that seek bar selector to less than 10 seconds - both selectors moving avoiding me to select less then 10 s, but for max duration it is not moving, and i can move it like i want, just Done button becomes not clickable. For users it is not very user friendly

**Comment by a914-gowtham** [comment]
Hi @Equin, Now I understand your issue. I felt the same from the beginning.  I've used [crystal-seekbar](https://github.com/syedowaisali/crystal-range-seekbar) as a trimming duration controller which has no option to help with our case.  I need to change the seekbar library. I will try to make the seekbar change Asap

**Comment by hsinha76** [comment]
Is this issue solved? @a914-gowtham
